wremnants/include/lowpu_utils.h

```diff
@@ -0,0 +1,168 @@
+#ifndef WREMNANTS_LOWPU_UTILS_H
+#define WREMNANTS_LOWPU_UTILS_H
+
+
+#include <eigen3/Eigen/Dense>
+#include <eigen3/unsupported/Eigen/CXX11/Tensor>
+#include "defines.h"
+#include "utils.h"
+
+namespace wrem {
+    
+Vec_i inverse(Vec_b tag) {
+
+    Vec_i res(tag.size(), 0);
+    for(unsigned int i = 0; i < res.size(); ++i) {
+        
+        if(tag[i] == 0) res[i] = 1;
+    }    
+    //res[i] = !tag[i];
+    return res;
+}
+
+
+Vec_i hasTriggerMatch_(const Vec_f& eta, const Vec_f& phi, const Vec_f& TrigObj_eta, const Vec_f& TrigObj_phi) {

```

Comment:
Is/does this need to be different from the existing hasTriggerMatch function?  (if so, please name it hasTriggerMatch_lowPU or similar to avoid confusion)

Reply:
It's a temporary function I use to debug these double-leg fired triggers. and muon norm issue. Later we can use the highPU triggerMatch function. For the time being changed the name to hasTriggerMatchLowPU